Source/Diagnostics/ParticleDiag/ParticleDiag.cpp

```diff
@@ -49,6 +49,15 @@ ParticleDiag::ParticleDiag(
                     m_plot_flags[existing_variable_names.at(var)] = 1;
                 }
             }
+            // Always write the position
+            // Note: for WARPX_DIM_RZ, no need to set the flag y: it is always written
```

Comment:
In RZ, I observed empirically that: 
- I **need** to set `m_plot_flags[pc->getParticleComps().at("x")] = 1;` (otherwise the Cartesian `x` position is not written correctly with openPMD)
- I **cannot** set `m_plot_flags[pc->getParticleComps().at("y")] = 1;` (otherwise I get an error in `at`, saying that this property does not exist), but the Cartesian `y` is automatically reconstructed and written anyway.

@ax3l Do you understand why `y` is automatically written, but not `x`? (i.e. why do I need to set the flag for `x`, but not for `y`)